autoload/syncopate.vim

```diff
@@ -89,6 +89,13 @@ function! s:InformUserAboutCopiedText(first, last)
   endif
 endfunction
 
+" wait to remove file
+function! s:WaitToRemoveFile(file)
+  sleep 1000m
```

Comment:
It looks like this adds a 1-second delay, which wasn't here before.  Is this really necessary?